SplunkRumWorkspace/SplunkRum/SplunkRum/SplunkRum.swift

```diff
@@ -335,4 +335,8 @@ var splunkRumInitializeCalledTime = Date()
         integrateWebViewWithBrowserRum(view: view)
     }
 
+    @objc public class func notificationTab(_ userInfo: UNNotificationResponse) {
```

Comment:
Where is this called from when users attach the library to their apps?

Reply:
**from didReceive delegate method.**
func userNotificationCenter(_ center: UNUserNotificationCenter, didReceive response: UNNotificationResponse, withCompletionHandler completionHandler: @escaping () -> Void) {
        let userInfo = response.notification.request.content.userInfo
       
        if let dict = userInfo as? [String: Any] {
            SplunkRum.notificationTab(response)
            
        }

        completionHandler()
    }

Reply:
It needs to happen automatically, users should not call anything from SplunkRum

Reply:
I tried with addObserver but was not able to get NotificationCenter events (didReceive response: UNNotificationResponse) in Splunk RUM.